getting_started/step_by_step/your_first_game.rst

```diff
@@ -865,49 +843,60 @@ Note that a new instance must be added to the scene using ``add_child()``.
 
     func _on_MobTimer_timeout():
         # Choose a random location on Path2D.
-        $MobPath/MobSpawnLocation.offset = randi()
+        var mob_spawn_location = get_node("MobPath/MobSpawnLocation");
+        mob_spawn_location.offset = randi()
+
         # Create a Mob instance and add it to the scene.
-        var mob = Mob.instance()
-        add_child(mob)
+        var mob_instance = _mob_scene.instance()
+        add_child(mob_instance)
```

Comment:
We're instancing a `mob` - why does this variable need to be expanded to `mob_instance`?

Reply:
For clarity's sake, but we can change this back.